Digisoft.Portal.Model/Models/Request/WifiFilterRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Digisoft.Portal.Model.Models.Request
+{
+    public class WifiFilterRequest
+    {
+        //public string? Name { get; set; }
```

Comment:
why we have commented code? also why can't we inherit BaseRequest model?

Reply:
I will remove it sir